src/grisp_nspi.erl

```diff
@@ -0,0 +1,38 @@
+% @private
+-module(grisp_nspi).
```

Comment:
The module name should end with `_drv`. We should use `grisp_spi_drv` here and replace the old unused driver instead.

Reply:
The same goes for the name of the C file.